Android.bp

```diff
@@ -33,7 +33,7 @@ mfx_cc_defaults {
     export_include_dirs: ["include"],
 
     include_dirs: [
-        "hardware/intel/external/minigbm-intel/cros_gralloc", // remove then minigbm gets Android.bp
+        "hardware/intel/external/minigbm/cros_gralloc", // remove then minigbm gets Android.bp
```

Comment:
Looks we don't have this folder hardware/intel/external/minigbm/cros_gralloc in celadon. What we have is hardware/intel/external/minigbm-intel.